tokio/src/runtime/basic_scheduler.rs

```diff
@@ -216,6 +210,220 @@ impl Spawner {
     }
 }
 
+impl<S> Queues<S>
+where
+    S: Schedule + 'static,
+{
+    pub(crate) const INITIAL_CAPACITY: usize = 64;
+
+    /// How often to check the remote queue first
+    pub(crate) const CHECK_REMOTE_INTERVAL: u8 = 13;
+
+    pub(crate) fn new() -> Self {
+        Self {
+            owned_tasks: UnsafeCell::new(task::OwnedList::new()),
+            local_queue: UnsafeCell::new(VecDeque::with_capacity(Self::INITIAL_CAPACITY)),
+            pending_drop: task::TransferStack::new(),
+            remote_queue: Mutex::new(RemoteQueue {
+                queue: VecDeque::with_capacity(Self::INITIAL_CAPACITY),
+                open: true,
+            }),
+        }
+    }
+
+    /// Add a new task to the scheduler.
+    ///
+    /// # Safety
+    ///
+    /// This *must* be called only from the thread that owns the scheduler.
+    pub(crate) unsafe fn add_task(&self, task: &Task<S>) {
+        (*self.owned_tasks.get()).insert(task);
+    }
+
+    /// Push a task to the local queue.
+    ///
+    /// # Safety
+    ///
+    /// This *must* be called only from the thread that owns the scheduler.
+    pub(crate) unsafe fn push_local(&self, task: Task<S>) {
+        (*self.local_queue.get()).push_back(task);
+    }
+
+    /// Push a task to the local queue.
```

Comment:
Release a task from the local queue!

Reply:
fixed in a01f5963215178b2a9042b996fe99e8a18987fe3, thanks for spotting!